webapp/channels/src/components/login/login.tsx

```diff
@@ -827,11 +840,13 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
                                     </span>
                                 </div>
                             )}
-                            {enableExternalSignup && (
+                            {true && (
```

Comment:
this condition must remain as it was before. It doesn't make any sense to hardcode a bool value in a condition.